buildx/working-with-buildx.md

```diff
@@ -14,24 +14,42 @@ multiple nodes concurrently.
 
 ## Install
 
-Docker Buildx is included in Docker Desktop and Docker Linux packages when
-installed using the [DEB or RPM packages](../engine/install/index.md).
+### Windows and macOS
 
-You can also download the latest `buildx` binary from the
-[Docker buildx](https://github.com/docker/buildx/releases/latest){:target="_blank" rel="noopener" class="_"} releases page
-on GitHub, copy it to `~/.docker/cli-plugins` folder with name
-`docker-buildx` and change the permission to execute:
+Docker Buildx is included in [Docker Desktop](https://www.docker.com/products/docker-desktop){:target="_blank" rel="noopener" class="_"}
+for Windows and macOS.
 
-```console
-$ chmod a+x ~/.docker/cli-plugins/docker-buildx
+### Linux packages
+
+Docker Linux packages also include Docker Buildx when installed using the
+[DEB or RPM packages](../engine/install/index.md).
+
+### Manual download
+
+You can also download the latest binary from the [releases page on GitHub](https://github.com/docker/buildx/releases/latest){:target="_blank" rel="noopener" class="_"}.
+
+Rename the relevant binary for your OS to `docker-buildx` and copy it to
+`$HOME/.docker/cli-plugins`.
+
+Or copy it into one of these folders for installing it system-wide:
+
+* `/usr/local/lib/docker/cli-plugins` OR `/usr/local/libexec/docker/cli-plugins`
+* `/usr/lib/docker/cli-plugins` OR `/usr/libexec/docker/cli-plugins`
+
+Might require to make it executable with `chmod +x`:
+
+```shell
+$ chmod +x ~/.docker/cli-plugins/docker-buildx
```

Comment:
Do we need manual instructions for Windows (and macOS) as well?

We should probably link this content from; https://docs.docker.com/engine/install/binaries/ as well

Reply:
Yes indeed. Maybe another page specific to plugins so we could use the same for others like compose, etc..

Reply:
Updated let me know what you think.